build/buildconstants/params_testground.go

```diff
@@ -140,3 +140,8 @@ const BootstrapPeerThreshold = 1
 const Eip155ChainId = 31415926
 
 var MinDealDuration, MaxDealDuration = policy.DealDurationBounds(0)
+
+const UnixfsChunkSize uint64 = 1 << 20
+const UnixfsLinksPerLevel = 1024
+
+var MinVerifiedDealSize = abi.NewStoragePower(256)
```

Comment:
```suggestion
```

these were removed in #12217 but restored again in your PR, so just remove here

also build/params_shared_vals.go needs UnixfsChunkSize, UnixfsLinksPerLevel removed and if you `git grep M..DealDuration -- build/` you'll see those vars that were also restored.

Reply:
Still needs addressing in this file